src/System.Text.RegularExpressions/src/System.Text.RegularExpressions.csproj

```diff
@@ -78,4 +78,8 @@
     <Reference Include="System.Reflection.Emit.Lightweight" />
     <Reference Include="System.Reflection.Primitives" />
   </ItemGroup>
+  <PropertyGroup>
+    <XUnitShowProgress>true</XUnitShowProgress>
+    <TestDisableParallelization>true</TestDisableParallelization>
```

Comment:
Historically I did this with `<XunitMaxThreads>1</XunitMaxThreads>` but maybe thats out of date.